modules/Bio/EnsEMBL/Production/Pipeline/GTF/DumpFile.pm

```diff
@@ -167,7 +145,7 @@ sub run {
 
   if ($gene){
     $self->info(sprintf "Checking GTF file %s", $out_file);
-    $self->_gene_pred_check($out_file) unless $eg;
+    $self->_gene_pred_check($out_file) unless $self->division ne 'vertebrates';
```

Comment:
```suggestion
    $self->_gene_pred_check($out_file) if $self->division eq 'vertebrates';
```

Reply:
Added it in the commit to restore checks on GTF checks. Thanks for that.